examples/apps/colorapp/README.md

```diff
@@ -303,6 +304,8 @@ In addition to the previously defined environment variables, you will also need
   
 ***Deploy services to ECS***
 
+> Note: Make sure you install [jq](https://stedolan.github.io/jq/) prior to running the ecs-colorapp.sh script
```

Comment:
It's not a bad idea to call this particular prerequisite out specifically. I'm fine with merging, but for consistency with the rest of the doc, would prefer that you remove the link portion (the parentheses and URL) since `[jq]` by itself will reference the correct URL in the links section at the end of the doc (using GitHub Flavored Markdown implicit links which leaves the main text less cluttered).

Reply:
That's neat! Made the change.

Reply:
Thanks!